mlflow/deployments/databricks/__init__.py

```diff
@@ -0,0 +1,44 @@
+from mlflow.deployments import BaseDeploymentClient
+
+
+class DatabricksDeploymentClient(BaseDeploymentClient):
+    def create_deployment(self, name, model_uri, flavor=None, config=None, endpoint=None):
+        raise NotImplementedError
+
+    def update_deployment(self, name, model_uri=None, flavor=None, config=None, endpoint=None):
+        raise NotImplementedError
+
+    def delete_deployment(self, name, config=None, endpoint=None):
```

Comment:
Curious what values would be in config or endpoint here? Is there an option to delete an endpoint referenced by name but not the entire named deployment?

Reply:
I don't think we need `xxx_deployment`.